### PR TITLE
[kube-prometheus-stack]: bump to 0.79.1

### PR DESCRIPTION
**Pull request by mjnagel**
#### What this PR does / why we need it

Updates to the latest prometheus operator CRDs (0.79.1) for kube-prometheus-stack.

#### Which issue this PR fixes

N/A

#### Special notes for your reviewer

CRDs copied in from prometheus-operator, straightforward changes.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by QuentinBisson — APPROVED** [review]

**Comment by mjnagel** [comment]
@QuentinBisson thanks - just pushed that update, not sure how I missed the script the first time.

**Comment by QuentinBisson** [comment]
Thank you :)

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 67.2.0
-appVersion: v0.79.0
+version: 67.3.0
```
Comment (mjnagel) [inline charts/kube-prometheus-stack/Chart.yaml:26]: Should I have just bumped the patch version here to correspond with the operator patch bump?

Comment (mjnagel) [reply charts/kube-prometheus-stack/Chart.yaml:26]:
```suggestion
version: 67.2.1
```

(realized when re-reviewing this that I just bumped the minor by default but that might not be right)

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**